pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -545,29 +546,88 @@ private void validatePoliciesReadOnlyAccess() {
         CompletableFuture<Boolean> isAuthorizedFuture;
 
         switch (operation) {
-            case LOOKUP: isAuthorizedFuture = canLookupAsync(topicName, role, authData);
+            case LOOKUP:
+            case GET_STATS:
+            case GET_SUBSCRIPTIONS:
+                isAuthorizedFuture = canLookupAsync(topicName, role, authData)
+                        .thenCombine(validateTenantAdminAccess(topicName.getTenant(), role, authData),
```

Comment:
for `LOOKUP`, the original behavior is checking `canLookupAsync`. Why do we need to check `validateTenantAdminAccess`?